plan.tex

```diff
@@ -57,15 +70,33 @@ \subsubsection{Middleware}
 we will have to do on IDF.
 
 \paragraph{Workflow} needs to be functioning at scale for DP0.2, ideally we should basic workflow
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also.
+We shall evaluate BPS as an option later in 2020 (DP-MW-M07).
+See \citeds{LDM-636}, \citeds{LDM-633}, \citeds{DMTN-123}.
+BPS translates the quantum graph to DAGMan for execution on HTCondor and submits the jobs.
+Most work has gone into the graph and execution.
+
+As part of our march toward a potential more DOE oriented Data Facility, BNL will be part of the the pre operations team to experiment with PanDA as an environment to monitor and control our processing jobs.
+This is a slightly parallel effort to construction attempting to take advantage of an existing set of tools for large scale job execution.
+In an ideal world the quantum graph translation of BPS would feed into a PanDA system to execute (retry etc) our jobs, this is still to be investigated.
+
+See also \secref{sec:in2p3}.
+
+
+
 
 \subsubsection{ Science Platform}
 The science platform and web services need to be deployed. In principle this is reasonable straight forward, an open issue may be configuring of the Portal aspect for the chosen  dataset(s).
 
 \subsubsection{ Pipelines }
 For DP0.2 we need a Gen3 version of the pipelines to process the dataset. This will have to run at scale for PDR2 or DC2. There may be several runs for quality purposes.
-Fractional FTE from the Pipelines will provide help in pipeline configuration, data repo preparation, workflow consulting, science verification, data model documenting, troubleshooting, and liaisoning.
+Fractional FTE from the Pipelines will provide help in pipeline configuration, data repo preparation, workflow consulting, science verification, data model documenting, troubleshooting, and liaising.
 {\bf Yusra will provide more info here.}
 
-\subsubsection{ IN2P3}
+\subsubsection{ IN2P3}\label{sec:in2p3}
 IN2P3 will contribute in Qserv and pipelines. {\bf Fabio will provide more information here.}
+They bring experience running Gen3 workflows. The real interest with IN2P3 is to run remote jobs thus emulating the eventual operational DRP runs. This may be difficult to achieve in FY21 but we should make it a milestone for FY22.\footnote{Tim, Fabio we should set a date for this}.
```

Comment:
In FY21 we might try to duplicate the same processing on IDF & IN2P3.

Reply:
ok